site/tests/universal/hubpage.test.ts

```diff
@@ -0,0 +1,34 @@
+import { expect, test } from "@playwright/test";
+
+test("Hub page should contain key elements", async ({ page }) => {
+  await page.goto("/hub");
+
+  await expect(
+    page.locator(
+      "h1 >> text=Interactive, data-driven blocks to use in your projects",
+    ),
+  ).toBeVisible();
+
+  await expect(
+    page.locator("text=All open-source and free to use"),
+  ).toBeVisible();
+
+  expect(
+    await page.locator('[data-testid="block-card"]').count(),
+  ).toBeGreaterThan(5);
```

Comment:
Optional, can be done in a follow-up: lets find one of the cards (Code?) and inspect it a bit further. We can make sure that it's clickable and the URL is of the right shape. We can also check elements like header, description, version and user. We don't have to match version precisely, just by a regexp. We can possibly escape some `data-testid`s by making the markup a bit more semantic (e.g. using header tag instead of div)

Reply:
I agree... I still think we need a way to target certain sections since we can have multiple semantic elements. A good thing to use would be classnames, but most classnames we have are generated and a bit cryptic

Reply:
Class names are an implementation detail, which is invisible to end users. It's better to interact with something more meaningful to humans such as visible text or a11y attributes.

If this is impossible, plan B is to use `data-testid`. We don't have to assign it to every element. If selecting by text is possible within a region, we can add `data-testid` to the region and then use text / a11y attributes within it.

Reply:
will address changes in a follow up

Reply:
Thanks, I will address this in a follow up

Reply:
Added tests for Code block card here => [6b37b72](https://github.com/blockprotocol/blockprotocol/pull/446/commits/6b37b72077bdbbe841be597dd28ab9f919eedcdd)